src/main/java/frc/robot/subsystems/climber/Climber.java

```diff
@@ -0,0 +1,150 @@
+ * be used for quick prototyping.
+ */
+public class Climber extends SubsystemBase {
+
+  // these Tunables are convenient when testing as they provide direct control of the subsystem's
+  // motor
+  private final TunableNumber leftMotorPower = new TunableNumber("Climber/leftPower", 0.0);
+  private final TunableNumber leftMotorCurrent = new TunableNumber("Climber/leftCurrent", 0.0);
+  private final TunableNumber leftMotorPosition = new TunableNumber("Climber/leftPosition", 0.0);
+
+  private final TunableNumber rightMotorPower = new TunableNumber("Climber/rightPower", 0.0);
+  private final TunableNumber rightMotorCurrent = new TunableNumber("Climber/rightCurrent", 0.0);
+  private final TunableNumber rightMotorPosition = new TunableNumber("Climber/rightPosition", 0.0);
+
+  private final SubsystemIOInputsAutoLogged inputs = new SubsystemIOInputsAutoLogged();
+  private ClimberIO io;
+
+  /**
+   * Create a new subsystem with its associated hardware interface object.
+   *
+   * @param io the hardware interface object for this subsystem
+   */
+  public Climber(ClimberIO io) {
+
+    this.io = io;
+
+    // Create a Shuffleboard tab for this subsystem if testing is enabled. Add additional indicators
+    // and controls as needed.
+    if (TESTING) {
+      ShuffleboardTab tab = Shuffleboard.getTab(CLIMBER);
+      tab.add(CLIMBER, this);
+    }
+
+    FaultReporter.getInstance().registerSystemCheck(CLIMBER, getSystemCheckCommand());
+  }
+
+  // FIXME: Update inputs needs to be fixed, currently have no idea what to do
+  /**
+   * The subsystem's periodic method needs to update and process the inputs from the hardware
+   * interface object.
+   */
+  @Override
+  public void periodic() {
+    io.updateInputs(inputs);
+    Logger.processInputs("Climber", inputs);
+
+    // when testing, set the motor power, current, or position based on the Tunables (if non-zero)
+    if (TESTING) {
+      if (motorPower.get() != 0) {
+        this.setMotorPower(motorPower.get());
+      }
+
+      if (motorCurrent.get() != 0) {
+        this.setMotorCurrent(motorCurrent.get());
+      }
+
+      if (motorPosition.get() != 0) {
+        this.setMotorPosition(motorPosition.get());
+      }
+    }
+  }
+
+  /**
+   * Set the motor power to the specified percentage of maximum power.
+   *
+   * @param power the percentage of maximum power to set the motor to
+   */
+  public void setLeftMotorPower(double power) {
+    io.setLeftMotorPower(power);
+  }
+
+  /**
+   * Set the motor current to the specified value in amps.
+   *
+   * @param power the current to set the motor to in amps
+   */
+  public void setLeftMotorCurrent(double power) {
+    io.setLeftMotorCurrent(power);
+  }
+
+  /**
+   * Set the motor position to the specified value in degrees.
+   *
+   * @param position the position to set the motor to in degrees
+   */
+  public void seLeftMotorPosition(double position) {
+    io.setLeftMotorPosition(position, POSITION_FEEDFORWARD);
```

Comment:
feedforward is a bit more complicated; probably not needed as a parameter and can be handled entirely in the hardware-specific class